nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -11,6 +11,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `python3` now defaults to Python 3.9, updated from Python 3.8.
 
+- Postgresql for NixOS service now defaults to v13.
```

Comment:
```suggestion
- PostgreSQL now defaults to major version 13.
```

Reply:
That's better. I just copied it from the 20.03 release notes.